packages/tradershub/src/modals/AddOrManageAccount/AddOrManageAccount.tsx

```diff
@@ -0,0 +1,88 @@
+    const { data: currencies, isLoading } = useCurrencies();
+    const [activeTab, setActiveTab] = useState<'CRYPTO' | 'FIAT'>(TabTypes[0]);
+
+    if (isLoading) return null;
+
+    const handleClose = () => {
+        setActiveTab(TabTypes[0]);
+        onClose();
+    };
+
+    return (
+        <ReactModal ariaHideApp={false} isOpen={isOpen} onRequestClose={handleClose} style={CUSTOM_STYLES}>
+            <div className='bg-system-light-primary-background lg:max-h-[717px] lg:max-w-[1040px] h-screen w-screen lg:rounded-default flex flex-col overflow-hidden'>
+                <div className='flex items-center justify-between w-full px-16 py-16 border-b border-solid lg:px-24 border-b-system-light-secondary-background'>
+                    <Text as='h3' className='text-lg' weight='bold'>
+                        Add or manage account
+                    </Text>
+                    <StandaloneXmarkBoldIcon className='cursor-pointer' onClick={handleClose} />
+                </div>
+                <Tabs
+                    TitleFontSize='sm'
+                    className='lg:w-[35%]'
+                    onChange={idx => setActiveTab(TabTypes[idx as keyof typeof TabTypes])}
+                    variant='secondary'
+                    wrapperClassName='flex justify-center'
+                >
+                    <Tab title='Cryptocurrencies' />
+                    <Tab title='Fiat currenciess' />
+                </Tabs>
+                {activeTab === TabTypes[0] && (
+                    <Fragment>
+                        <div className='flex flex-col items-center mt-24'>
+                            <Text as='p' className='text-lg' weight='bold'>
+                                Choose your preferred cryptocurrency
+                            </Text>
+                            <Text as='p' className='text-sm'>
+                                You can open an account for each cryptocurrency.
+                            </Text>
+                        </div>
+                        <CurrenciesForm currencies={currencies?.CRYPTO ?? []} />
+                    </Fragment>
+                )}
+                {activeTab === TabTypes[1] && (
+                    <Fragment>
+                        <div className='flex flex-col items-center mt-24'>
+                            <Text as='p' className='text-lg' weight='bold'>
+                                Change your currency
+                            </Text>
+                            <Text as='p' className='text-sm'>
+                                Choose the currency you would like to trade with.
+                            </Text>
+                        </div>
+                        <CurrenciesForm currencies={currencies?.FIAT ?? []} submitButtonLabel='Change currency' />
+                    </Fragment>
+                )}
+            </div>
+        </ReactModal>
+    );
+};
+
+export default memo(AddOrManageAccount);
```

Comment:
@shahzaib-deriv what is the benefit to wrap it by memo? React.memo is expensive operation and we don't need to use it everywhere. Also, you wrapped this component with memo, but in CurrencySwitcher component write like this:

![image](https://github.com/binary-com/deriv-app/assets/120570511/c6a95143-b8ca-4702-9b96-db293fb0914b)
 
In result you pass new callback to onClose for EVERY render, but memo doesn't save you from re-render if you pass new function to props... Or delete this HOC memo, or wrap function which you pass to onClose with useCallback. My opinion, we have to delete this memo, because useCallback is expensive too. You have to measure how often this component rerenders, maybe it works fine without it and rerenders only when new props come

Reply:
@sergei-deriv thanks for pointing it out. Lemme update the `onClose` as as callback. 
This component is being rerender a lot of times as you mentioned like `onClose` being created on each render. However `memo` is not an expensive operation. React team internally in Facebook uses it a lot make components stable and prevent unnecessary renders.

Reply:
@shahzaib-deriv just one question. If this is so cheap and we need always use it, in this case why react team in Facebook didn't make this wrapper by default? Or why in the old react we could extends from Component or from PureComponent? Looks like react team knows we don't need to use memo for every components... One more time, if you think we can avoid a lot of re-renders, then ok, use it!